LondonBritishLibrary/orient/BLorient581.xml

```diff
@@ -54,47 +263,92 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 </support>
                                 <extent>
                                     <measure unit="leaf">206</measure>
-                                    <dimensions type="outer" unit="mm">
-                                        <height>232</height>
-                                        <width>203</width>
+                                    <dimensions type="outer" unit="in">
+                                        <height>9.125</height>
+                                        <width>8</width>
                                     </dimensions>
                                 </extent>
-                                
-                                
+
+
                             </supportDesc>
-                            
+
                             <layoutDesc>
                                 <layout columns=" 3" writtenLines="22" cert="high"/>
                             </layoutDesc>
                         </objectDesc>
-                        
+
                         <handDesc>
-                            
+
                             <handNote script="Ethiopic" xml:id="h1">
                                 <desc>The manuscript is neatly written of the earlier part of the <date>18th century</date>
                                 </desc>
-                                
+
                                 <date notAfter="1700" notAfter-custom="1800"/>
-                                
+
                             </handNote>
-                            
-                        </handDesc>   
+
+                        </handDesc>
                         <additions>
-                            
+<list>
+
+<item xml:id="a1">
+<locus target="#206r"/>
+<desc type="GuestText">Wazema</desc>
+<q xml:lang="gez"><seg part="I">እምነገደ፡ አዳም፡ ወሔዋን፡ አልቦ፡ ከማኪ፡ በፈሊጠ፡ ኵሉ፡ ምግባት፡</seg></q>
+</item>
+
+<item xml:id="a2">
+<locus target="#206r"/>
+<desc type="RecordTransaction">A note in Amharic concerning a land transaction.</desc>
+<q xml:lang="am"></q>
+</item>
+
+<item xml:id="a3">
+<locus target="#206v"/>
+<desc type="MagicText">A magical prayer against the disease <foreign xml:lang="gez">ፈንጻጻ፡</foreign>.</desc>
```

Comment:
I guess we use "magic" rather than "magical"

Reply:
Will try to remember